libs/ui-watt/src/lib/components/input/timepicker/watt-timepicker.component.ts

```diff
@@ -43,6 +52,28 @@ import { WattRangeInputService } from '../shared/watt-range-input.service';
 const hoursMinutesFormat = 'HH:MM';
 const hoursMinutesPlaceholder = 'HH:MM';
 
+// Show slider initially as "00:00 - 23:59"
+const initialSliderValue: WattSliderValue = { min: 0, max: 1439 };
+
+/** Converts string time format (HH:MM) to number of minutes. */
+function timeToMinutes(value: string) {
+  const [hours, minutes] = value.split(':');
+  return parseInt(hours) * 60 + parseInt(minutes);
```

Comment:
suggestion: use the Number primitive to convert string to a number.

```suggestion
  return Number(hours) * 60 + Number(minutes);
```

Reply:
4d3d112476f8929e6f1780198f51ccbdd88f2502